src/kind_lab_to_nwb/arc_ecephys_2024/insert_session.py

```diff
@@ -112,16 +114,19 @@ def print_tables(nwbfile_path):
         print(sgc.CameraDevice & [{"camera_name": camera_name} for camera_name in camera_names], file=f)
         print("=== Task Epoch ===", file=f)
         print(sgc.TaskEpoch(), file=f)
+        print("=== SensorData ===", file=f)
+        print(sgc.SensorData(), file=f)
 
 
 def main():
     nwbfile_path = Path(
-        "/media/alessandra/HD2/kind_lab_conversion_nwb/Spyglass/raw/sub_Rat_1716-ses_Baseline_tone_flash_hab.nwb"
+        "/media/alessandra/HD2/kind_lab_conversion_nwb/Spyglass/raw/sub_Rat_1717-ses_Baseline_tone_flash_hab.nwb"
     )
 
     clean_db_entry(nwbfile_path)
 
     sgi.insert_sessions(str(nwbfile_path), rollback_on_fail=True, raise_err=True)
+    sgc.SensorData.populate()
```

Comment:
Looks like you shouldn't need to populate this separately after this fix: https://github.com/LorenFrankLab/spyglass/pull/1281

Have you tried using the latest dev version of spyglass?